content/pages/faq.md

```diff
@@ -0,0 +1,52 @@
+Title: FAQ
+Date: 2015-01-28 12:00
+Category: pages
+Summary: WiCS Frequenctly Asked Questions
+
+Here you will find answers to questions we commonly receive. If you have a 
+question not listed below, please do not hesitate to reach out to us!
+
+# Why do you sometimes hold women-only events? #
```

Comment:
You can probably make this subheader size (##), as opposed to a full page header. I imagine the questions look huge!